packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4794,9 +4794,10 @@ export class Base64Decode extends Op {
 		}
 	}
 
-	execute(stack: TEALStack): void {
+	execute(stack: TEALStack): number {
 		this.assertMinStackLen(stack, 1, this.line);
 		const last = this.assertBytes(stack.pop(), this.line);
+		const cost = 1 + Math.ceil(last.length / 16);
```

Comment:
Please add comment to explain compute `cost` formula.

Reply:
Done